packages/coq/coq.8.8.2/opam

```diff
@@ -21,7 +21,7 @@ build: [
     "-docdir" doc
     "-libdir" "%{lib}%/coq"
     "-datadir" "%{share}%/coq"
-    "-camlp5dir" "%{lib}%/camlp5"
+    "-camlp5dir" "%{camlp5:lib}%"
```

Comment:
why is the reason behind this change? Also why change it for almost all versions but the latests

Reply:
I see. Thanks